device-types/Dell/PowerEdge-R750xs.yml

```diff
@@ -0,0 +1,81 @@
+---
+manufacturer: Dell
+model: PowerEdge R750xs
+slug: dell-poweredge-r750xs
+u_height: 2
+is_full_depth: true
+front_image: true
+rear_image: true
+comments: |
+  [PowerEdge R750xs](https://dl.dell.com/content/manual49765834-dell-emc-poweredge-r750xs-manuel-d-installation-et-de-maintenance.pdf?language=fr-fr)
+rear-ports:
+  - name: Rear VGA
+    type: other
+  - name: Rear USB 2
+    type: other
+  - name: Rear USB 3
+    type: other
+  - name: iDrac
+    type: other
+front-ports:
+  - name: Front VGA
+    type: other
+    rear_port: Rear VGA
+  - name: Front USB 2
+    type: other
+    rear_port: Rear USB 2
+  - name: Front USB micro-AB
+    type: other
+    rear_port: iDrac
+power-ports:
+  - name: Power 1
+    type: iec-60320-c14
+  - name: Power 2
+    type: iec-60320-c14
+interfaces:
+  - name: Gig-E 1
+    type: 1000base-t
+  - name: Gig-E 2
+    type: 1000base-t
+  - name: iDRAC
+    type: 1000base-t
+    mgmt_only: true
+module-bays:
+  - name: AHCI controller in SL 10
+    position: AHCI.SL.10-1
+    label: BOSS-S2
+  - name: Backplane 1 on Connector 0 of RAID Controller in Slot 1
+    position: Internal.0-1:RAID.Slot.1-1
+    label: BP_PSV 0:1
+  - name: Battery on RAID Controller in Slot 1
+    position: Integrated.1:RAID.Slot.1-1
+  - name: CPU 1
+    position: CPU.Socket.1
+    label: Intel(R) Xeon(R) Gold 5320 CPU @ 2.20GHz
+  - name: CPU 2
+    position: CPU.Socket.2
+    label: Intel(R) Xeon(R) Gold 5320 CPU @ 2.20GHz
+  - name: DIMM A1
+    position: DIMM.Socket.A1
+    label: DDR4 DIMM
+  - name: DIMM A2
+    position: DIMM.Socket.A2
+    label: DDR4 DIMM
+  - name: DIMM A3
+    position: DIMM.Socket.A3
+    label: DDR4 DIMM
+  - name: DIMM A4
+    position: DIMM.Socket.A4
+    label: DDR4 DIMM
+  - name: DIMM B1
+    position: DIMM.Socket.B1
+    label: DDR4 DIMM
+  - name: DIMM B2
+    position: DIMM.Socket.B2
+    label: DDR4 DIMM
+  - name: DIMM B3
+    position: DIMM.Socket.B3
+    label: DDR4 DIMM
+  - name: DIMM B4
+    position: DIMM.Socket.B4
+    label: DDR4 DIMM
```

Comment:
Unfortunately these should not be modelled in NetBox at this time. Please remove them.